### DISCUSSION TITLE
Design creations for website on Bootstrap

### DISCUSSION THREAD

**Discussion by geener99** [twbs/bootstrap#39625] [Q&A]
I tried to find some tutorials or examples how create web design if i going to do website on bootstrap but alone what i found is bootstrap ui kit for figma https://www.figma.com/community/file/876022745968684318 and one silent video how man create web design for his bootstrap course https://www.youtube.com/watch?v=VKEthyhc7GM

Can someone describe me how creating design layout for website what will build on bootstrap. What i need considered. Because for now I have a feeling that all use bootstrap always without design layout and create website immediately in code. I think this approach will suit for admin panel but simple site not much. The same goes for the uikit that I found

In general i ask describe me what approaches exist for design for bootstrap websites in production

Thank you in advance

**Comment by Albert7866** [comment]
It’s true that many [developers ](https://glorymoto.com/collection/leather-motorcycle-shoes/)jump straight into coding with Bootstrap, especially for admin panels or simple sites. However, creating a design layout first can ensure a more polished result. Typically, designers start with wireframes or mockups (using tools like Figma) and consider Bootstrap's grid system, components, and breakpoints for responsiveness. A solid approach includes designing with Bootstrap's limitations and strengths in mind, then translating that into clean code. Balancing design and development early on will lead to better outcomes.
